lib/trusted-apps.js

```diff
@@ -90,6 +93,29 @@ function middleware(req, res, next) {
   next();
 }
 
+function middlewareForKoa(ctx) {
+
+  const url = getRequestUrl(ctx.req);
+  const certificate = getReqHeaderValue(ctx.req, HEADER_CERTIFICATE);
+  const providerId = getReqHeaderValue(ctx.req, HEADER_PROVIDER_ID);
+  const signature = getReqHeaderValue(ctx.req, HEADER_SIGNATURE);
+
+  ctx.assert(certificate, 403, "No trusted apps certificate found");
```

Comment:
If the middleware functions are going to have different implementations could they at least have similar behaviour? e.g. the Express middleware will allow the request through if there is no certificate